runTestsLocally

```diff
@@ -0,0 +1 @@
+rm -rf .history/ && cd src && wollok test && cd .. && WOLLOK_DEV_ROOT=../wollok wollok test -dontValidate
```

Comment:
Doesn't this require a particular implementation of Wollok to be on that path? Maybe we shouldn't commit this.

Reply:
Such a pity, I'll have to put it somewhere else (and possibly forget it)

Reply:
If it's a big deal we can keep it. But maybe is as easy as moving this script to wollok-xtend. In ts we have it as a building pipeline script

Reply:
Yes, running sanity test is part of the CI pipeline, but if you are in wolllok-language and make a change that breaks wollok-xtext implementation,

a) if it's intentional => ok
b) if it's not => at least you should know somehow

I believe we should provide scripts to check this for both implementations, and alert this situation.